### PR TITLE
Feat(private scanning): Add private scanning features

### PR DESCRIPTION
**Pull request by pabloperezj**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/VirusTotal/Playbooks/Detonate_Private_File_-_VirusTotal_V3.yml [hunk 1]
```diff
@@ -0,0 +1,389 @@
+id: Detonate Private File - VirusTotal (API v3)
+version: -1
+name: Detonate Private File - VirusTotal (API v3)
+description: Detonate a private file through VirusTotal (API v3)
```
Comment (danipv) [inline Packs/VirusTotal/Playbooks/Detonate_Private_File_-_VirusTotal_V3.yml:4]:
```suggestion
description: Detonate a private file through VirusTotal Private Scanning service
```

File: Packs/VirusTotal/Playbooks/Detonate_Private_File_-_VirusTotal_V3.yml [hunk 2]
```diff
@@ -0,0 +1,389 @@
+id: Detonate Private File - VirusTotal (API v3)
+version: -1
+name: Detonate Private File - VirusTotal (API v3)
```
Comment (danipv) [inline Packs/VirusTotal/Playbooks/Detonate_Private_File_-_VirusTotal_V3.yml:3]:
```suggestion
name: Detonate Private File - VirusTotal Private Scanning
```

File: Packs/VirusTotal/Playbooks/Detonate_Private_File_-_VirusTotal_V3_README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+Detonate a private file through VirusTotal (API v3)
```
Comment (danipv) [inline Packs/VirusTotal/Playbooks/Detonate_Private_File_-_VirusTotal_V3_README.md:1]:
```suggestion
Detonate a private file through VirusTotal Private Scanning service

See files through the eyes of VirusTotal without uploading them to the main threat corpus, keeping them entirely private. Static, dynamic, network and similarity analysis included, as well as automated threat intel enrichment, but NOT multi-antivirus analysis.

More information: https://support.virustotal.com/hc/en-us/articles/8516907790749-Private-Scanning

Get Private Scanning: https://www.virustotal.com/gui/contact-us
```

File: Packs/VirusTotal/Integrations/VirusTotalV3/README.md [hunk 4]
```diff
@@ -2158,3 +2158,382 @@ Retrieves result of the last Sigma analysis.
 >|MatchContext|RuleLevel|RuleDescription|RuleSource|RuleTitle|RuleId|RuleAuthor|
 >|---|---|---|---|---|---|---|
 >| $EventID: '1117' | high | Detects all actions taken by Windows Defender malware detection engines | Sigma Integrated Rule Set (GitHub) | Windows Defender Threat Detected | 693c36f61ac022fd66354b440464f490058c22b984ba1bef05ca246aba210ed1 | Ján Trenčanský |
+
+
+### vt-privatescanning-file
+
+***
+Checks the file reputation of the specified private hash.
```
Comment (danipv) [inline Packs/VirusTotal/Integrations/VirusTotalV3/README.md:2166]:
```suggestion
Checks the file reputation of the specified private hash.

See files through the eyes of VirusTotal without uploading them to the main threat corpus, keeping them entirely private. Static, dynamic, network and similarity analysis included, as well as automated threat intel enrichment, but NOT multi-antivirus analysis.
```

File: Packs/VirusTotal/pack_metadata.json [hunk 5]
```diff
@@ -2,7 +2,7 @@
     "name": "VirusTotal",
     "description": "Analyze suspicious hashes, URLs, domains and IP addresses",
     "support": "partner",
-    "currentVersion": "2.5.2",
+    "currentVersion": "2.5.3",
```
[the remark this answers is not in the document]
Comment (pabloperezj) [reply Packs/VirusTotal/pack_metadata.json:5]: Done. 2.6.0

**Review by danipv — APPROVED** [review]
LGTM

**Pull request closed without merging**